recipe-generator/src/main/resources/templates/ts/enum-types.liquid

```diff
@@ -5,4 +5,6 @@ export class {{enum.name}} {
     private value;
     private constructor();
     getValue(): string;
+    toJSON(): any;
```

Comment:
shouldn't it return `string`? and the next method accept a string param?

Reply:
`JSON.stringify` automatically called `toJSON()` and expects a "json object", which it turns into a string.